pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -537,7 +531,18 @@ def get_vaccination_data(read_data=dd.defaultDict['read_data'],
         The sanitizing threshold will be defined by the age group-specific
         average on the corresponding vaccination ratios on county and federal
         state level.
-    @param moving_average 0 [Default] or Number>0. Defines the number of days for which a centered moving average is computed.
+    @param read_data [Currently not used] True or False. Defines if data is read from file or downloaded.
+        Here Data is always downloaded from the internet.
+    @param file_format File format which is used for writing the data. Default defined in defaultDict.
+    @param out_folder Folder where data is written to. Default defined in defaultDict.
+    @param no_raw True or False. Defines if unchanged raw data is saved or not. Default defined in defaultDict.
+    @param start_date Date of first date in dataframe. Default defined in defaultDict.
+    @param end_date Date of last date in dataframe. Default defined in defaultDict.
+    @param impute_dates True or False. Defines if values for dates without new information are imputed. 
+        Here Dates are always imputed so False changes nothing.
+    @param moving_average Integers >=0. Applies an 'moving_average'-days moving average on all time series
```

Comment:
moving_average is not in the parameter list above although it should be?!

Reply:
I don't see what you mean.